src/create-twilio-function/create-files.js

```diff
@@ -74,10 +93,44 @@ function createNvmrcFile(pathName) {
   return createFile(fullPath, content);
 }
 
+function createTsconfigFile(pathName) {
+  const fullPath = path.join(pathName, 'tsconfig.json');
+  return createFile(
+    fullPath,
+    JSON.stringify(
+      {
+        compilerOptions: {
+          target: 'es5',
+          module: 'commonjs',
+          strict: true,
+          esModuleInterop: true,
+          outDir: 'dist',
+          skipLibCheck: true,
```

Comment:
Does that just mean it ignores if dependencies have errors? or is this impacting the checks of the actual code?

Reply:
This is about dependencies. In particular, TypeScript always seems to have a problem on my machine with `WebGL2RenderingContext`. I'm never doing anything with `WebGL2RenderingContext` so I don't know what's going on.

This skips type checking dependencies like that, and makes sense because there's nothing we can do about those dependencies anyway.